pkg/validator/validator_test.go

```diff
@@ -119,13 +124,6 @@ func TestStateValidation(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-
-			tx := db.MustBegin()
-			defer tx.Rollback()
-			err = validator.ValidateReferentialIntegrity(tx, i)
```

Comment:
Are we still testing this elsewhere?

Reply:
Yes, sorry I moved it to the ref integrity tests, but accidentally moved that commit to another branch. Fixed.